crux-agent/app/api/routers/jobs.py

```
@@ -164,4 +347,62 @@ async def cancel_job(
+            "management": {
+                "summarization_threshold": settings.summarization_threshold,
+                "response_reserve": settings.response_reserve
+            },
+            "fallback_limit": 50000  # Used when no specific model match is found
+        }
+    except Exception as e:
+        logger.error(f"Error getting context limits: {e}")
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"Failed to get context limits: {str(e)}",
+        )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)